platform_channels/lib/src/event_channel_demo.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:platform_channels/src/accelerometer_event_channel.dart';
+
+/// The widget demonstrates how to use [EventChannel] to listen continuous values
+/// of Accelerometer Sensor from platform.
+///
+/// The widget uses a [StreamBuilder] to rebuild it's descendant whenever it
+/// listens a new value from the Accelerometer.getReadings() stream. It also
+/// has three [Text] widgets to display the value of [AccelerometerReadings.x],
+/// [AccelerometerReadings.y], and [AccelerometerReadings.z] respectively.
+class EventChannelDemo extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final textStyle = Theme.of(context).textTheme.headline5;
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('EventChannel Demo'),
+      ),
+      body: StreamBuilder<AccelerometerReadings>(
+        stream: Accelerometer.getReadings(),
+        builder: (context, snapshot) {
+          if (snapshot.hasData) {
+            return Center(
+              child: Column(
```

Comment:
Any suggestions for improving UI are welcome. We can also have three `Contianer`s which change properties when a value changes. But since the sample is for learning purpose I kept it simple enough.